plugins/kubernetes/app/models/kubernetes/role_verifier.rb

```diff
@@ -146,11 +147,22 @@ def verify_project_and_role_consistent
 
     def verify_stateful_set_service_consistent
       return unless service = @elements.detect { |t| t[:kind] == "Service" }
-      return unless set = @elements.detect { |t| t[:kind] == "StatefulSet" }
+      return unless set = find_stateful_set
       return if set.dig(:spec, :serviceName) == service.dig(:metadata, :name)
       @errors << "Service metadata.name and StatefulSet spec.serviceName must be consistent"
     end
 
+    def verify_stateful_set_restart_policy
+      return unless set = find_stateful_set
+      return if set.dig(:spec, :updateStrategy)
```

Comment:
I am not sure this matches our yml file structure. https://github.com/zendesk/help-center-memcached-k8s/blob/a0790e62a4cbda6fc52172f1ae79154d6d385679/kubernetes/memcached.yml#L20

Honestly, I rather have @jacobat and @dasch opinion here. Not sure if we use `OnDelete` or `RollingUpdate `

Reply:
It looks :+1: to me.

Reply:
the default is OnDelete, that's why I'm calling it out explicitly